Source/WebCore/Modules/WebGPU/GPUDeviceError.idl

```diff
@@ -26,7 +26,7 @@
 // https://gpuweb.github.io/gpuweb/#ref-for-gpudevice⑤⓪
 
 // https://bugs.webkit.org/show_bug.cgi?id=232548 This shouldn't need to be here.
-typedef (GPUOutOfMemoryError or GPUValidationError) GPUError;
+typedef (GPUOutOfMemoryError or GPUValidationError or GPUInternalError) GPUError;
```

Comment:
The [spec](https://gpuweb.github.io/gpuweb/#gpuerror) says

```
interface GPUError {
    readonly attribute DOMString message;
};
```

Are we sure our IDL is right?

Reply:
GPUError is a base type according to the specification and we could / probably should implement this as done in the specification.  I think the result is the same however as the base type is never used in an error. To avoid refactoring GPUOutOfMemoryError or GPUValidationError I used the existing pattern which uses a typedef.

Reply:
I filed https://bugs.webkit.org/show_bug.cgi?id=253744 to track this and will add a FIXME here